packages/contracts/test/voting/token-voting.ts

```diff
@@ -915,8 +919,12 @@ describe('TokenVoting', function () {
       it('cannot early execute', async () => {
         await advanceIntoVoteTime(startDate, endDate);
 
-        await governanceErc20Mock.mock.getPastVotes.returns(51);
-        await voting.vote(id, VoteOption.Yes, false);
+        await voting.connect(signers[0]).vote(id, VoteOption.Yes, false);
+        await voting.connect(signers[1]).vote(id, VoteOption.Yes, false);
+        await voting.connect(signers[2]).vote(id, VoteOption.Yes, false);
+        await voting.connect(signers[3]).vote(id, VoteOption.Yes, false);
+        await voting.connect(signers[4]).vote(id, VoteOption.Yes, false);
+        await voting.connect(signers[5]).vote(id, VoteOption.Yes, false);
 
```

Comment:
could also be optimized with a loop and `Promise.all`

Reply:
Improved in bbc468ded7f531d27ce685c55d3dd166acc20bef